app/lib/training_route_manager.rb

```diff
@@ -9,8 +9,19 @@ def requires_placement_details?
     feature_enabled?("routes.provider_led_postgrad") && provider_led_postgrad?
   end
 
+  def award_type
+    case trainee.training_route
```

Comment:
There's a method `training_route` already in `TrainingRouteManager` we can use here.